### DISCUSSION TITLE
How to trigger Suspense when fetching from atomWithStorage?

### DISCUSSION THREAD

**Discussion by enzomanuelmangano** [pmndrs/jotai#1968] [Bug report]
First of all, thank you very much for this **outstanding** State Management and all the support. 

The problem I am experiencing is that by accessing via **useAtomValue** an atom defined with **atomWithStorage**, the atom is first initialized and then has its actual value. 

I am wondering if this is the behavior I should expect. I honestly expected Suspense to be triggered until the atom has the actual value (already fetched from the store). 

What am I missing?

**Comment by dai-shi** [comment]
#1958 will fix it.
